### PR TITLE
adding LanczosLowNoice

### PR DESCRIPTION
**Pull request by reula**
I added Lanczos and SuperLanczos in the same structure, for SuperL is just the order 4 of Lanczos

### CODE REVIEW

File: src/periodic_operators.jl [hunk 1]
```diff
@@ -527,6 +527,150 @@ function periodic_derivative_coefficients(source::Holoborodko2008, derivative_or
...
+                ])
+                upper_coef = -lower_coef
+                central_coef = T(0)
+            else
+                method_exists = false
+            end
+        else
+            method_exists = false
+        end
+    elseif derivative_order == 2
```
Comment (ranocha) [inline src/periodic_operators.jl:621]: Could you please remove this branch containing only code that is commented out?

**Review by ranocha — CHANGES_REQUESTED** [review]
Thanks a lot! I will start tests in the cloud.

File: src/periodic_operators.jl [hunk 2]
```diff
@@ -527,6 +527,150 @@ function periodic_derivative_coefficients(source::Holoborodko2008, derivative_or
     PeriodicDerivativeCoefficients(lower_coef, central_coef, upper_coef, mode, derivative_order, accuracy_order, source)
 end
 
+"""
+    LanczosLowNoice()
```
Comment (ranocha) [inline src/periodic_operators.jl:531]: Could you please fix the typo "noice" -> "noise" everywhere in the code?

File: src/periodic_operators.jl (hunk 1 above)
Comment (reula) [reply src/periodic_operators.jl:621]: Done

File: src/periodic_operators.jl (hunk 2 above)
Comment (reula) [reply src/periodic_operators.jl]: Done

File: src/periodic_operators.jl [hunk 3]
```diff
@@ -527,6 +527,104 @@ function periodic_derivative_coefficients(source::Holoborodko2008, derivative_or
...
+                ])
+                upper_coef = -lower_coef
+                central_coef = T(0)
+            else
+                method_exists = false
+            end
+        else
+            method_exists = false
+        end
+
```
Comment (ranocha) [inline src/periodic_operators.jl:621]:
```suggestion
        end
    end
```

**Comment by ranocha** [comment]
Thanks a lot for your contribution! I could not push to the `main` branch of your fork, so I have taken your commits and fixed the remaining issues in another PR.

**Pull request closed without merging**